src/compass/utils/lut.py

```diff
@@ -23,7 +30,19 @@ def compute_geocoding_correction_luts(burst, rg_step=200, az_step=0.25):
     az_bistatic = burst.bistatic_delay(range_step=rg_step, az_step=az_step)
     rg_doppler = burst.geometrical_and_steering_doppler(range_step=rg_step, az_step=az_step)
 
+    if path_dem is None:
+        # Skip the FM mismatch rate calculation
+        az_fm_mismatch = None
+    else:
+        az_fm_mismatch = burst.az_fm_rate_mismatch_mitigation(path_dem,
+                                                              path_scratch,
+                                                              rg_step=rg_step,
+                                                              az_step=az_step)
+
     rg_lut = rg_doppler
     az_lut = az_bistatic.data
 
-    return rg_lut, az_lut
\ No newline at end of file
+    if not az_fm_mismatch is None:
+        az_lut += az_fm_mismatch.data
```

Comment:
let's coordinate with @yunjunz and make sure the signs are consistent with ETAD and we can add the two.

Reply:
@yunjunz The difference between our calculation and the ETAD data was small. So the sign of our calculation and the ETAD data would be the same. Therefore, I think the sign to apply our azimuth FM rate mismatch mitigation has to be the same as we do with ETAD data.

Wound you be able to confirm if we are "adding" the correction when we are using ETAD?

Reply:
Please @yunjunz, confirm what @seongsujeong has mentioned above. Also to my understanding, to be consistent with ETAD, we should add the two corrections

Reply:
My testings with `ISCE2/topsApp.py` shows the `geometrical offset - ETAD offset` gives the correct result (fewer phase artifacts due to SAR processing effects), thus, if our `az_lut` here is consistent with isce2, the sign should be `-=`.

Reply:
Regarding https://github.com/opera-adt/COMPASS/pull/72#discussion_r1047826363, yes, we should include all 3 components in the final offsets.